components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2099,6 +2099,7 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
     async listBilledUsage(
         ctx: TraceContext,
         attributionId: string,
+        startedTimeOrder: number,
```

Comment:
I didn't use the `SortOrder` type in the server because TypeScript kept complaining that it's expecting something else.

Reply:
Here's a comparison that shows how it can be used:

https://github.com/gitpod-io/gitpod/compare/laushinka/incorporate-reverse-sorting-11691...af/temp

Does that work for you? Perhaps the TS error was due to not rebuilding after changing the `gitpod-protocol` to add the enum?

Reply:
Aha `startedTimeOrder as number` is the key! Thank you, @andrew-farries!